dags/requirements.txt

```diff
@@ -1,4 +1,4 @@
---constraint /usr/local/airflow/dags/requirements-constraints.txt
+#--constraint /usr/local/airflow/dags/requirements-constraints.txt
```

Comment:
I'm not sure why this change is needed - found the source at https://github.com/US-GHG-Center/ghgc-data-airflow/pull/27 but I just want to double check that we're still pulling in the constraints elsewhere, or that we're certain that they're no longer needed.

Reply:
I recall removing the constraint because it caused an issue with a specific version (possibly rasterio) which makes the scheduler to fail, but this change seems to not have any side effect.